java/src/jmri/jmrit/operations/rollingstock/cars/JmritOperationsCarsBundle.properties

```diff
@@ -21,6 +21,9 @@ FD                  = FD
 ReturnWhenEmpty     = Return When Empty
 RWELocation			= RWE Location
 RWELoad             = RWE Load
+ReturnWhenLoaded    = Return When Loaded
+RWLLocation			= RWL Location
```

Comment:
There are TABS on this line, instead of spaces. No big deal and no reason to hold this PR. But I mention so you know.

Reply:
@danielb987 Thanks Daniel.